spec/acceptance/mysql_backup_spec.rb

```diff
@@ -29,55 +27,48 @@ class { 'mysql::server::backup':
         }
     MANIFEST
     it 'when configuring mysql backups' do
-      idempotent_apply(default, pp, {})
+      idempotent_apply(pp)
     end
   end
 
-  describe 'mysqlbackup.sh' do
+  describe 'mysqlbackup.sh', if: Gem::Version.new(mysql_version) < Gem::Version.new('5.7.0') do
```

Comment:
Instead of putting the skip at this level, you might consider putting a `skip` in a `before` block using this condition as in [this change](https://github.com/puppetlabs/puppetlabs-chocolatey/pull/149/files#diff-983b68f4d1d6ab54a11cc731866cf92fR17).
The advantage gained is that you can then see in the CI output exactly which it blocks were skipped, which gives increased visibility into exactly how many tests are not being run on a give CI configuration and for what reason, while still reducing the verbosity and repetitiveness of the previous pattern of the `unless` block in every test.
We moved to this pattern recently because our inability to answer that question was causing issues for us.